### DISCUSSION TITLE
Next project to do

### DISCUSSION THREAD

**Discussion by yrncollo** [yrncollo/cpp-programming#29] [General]
> Hey @yrncollo, should we start working on other systems as well simultaneously or we should first finish one, and then only we should move to another?

Hey @dishamodi0910 So which project are you suggesting we jump into?

**Comment by dishamodi0910** [comment]
Hey @yrncollo, I think we should go by the sequence mentioned in the readme, so the airline reservation system can be there the next one, you can share your opinions😃.

**Reply by yrncollo** [reply]
Awesome

**Reply by dishamodi0910** [reply]
If possible, you can create it's base as it was created in banking management system by including things in readme.
And if you won't mind, should I do?

**Reply by yrncollo** [reply]
Yeah yeah you can do... make sure you are on a new branch.... then follow the directory structure just like the banking management so that we can code in the /src directory...

**Reply by dishamodi0910** [reply]
Yeahhh will take care of that!

**Reply by yrncollo** [reply]
Awesome lemme wait for your PR
